ci/jenkins/jobs/macros.yaml

```diff
@@ -133,6 +133,33 @@
           rm -rf "${WORKSPACE}/jenkins"
           echo "=== Cleanup cluster ${cluster} succeeded ==="
 
+- builder:
+    name: builder-eks-cluster-cleanup
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+
+          cluster="${CLUSTERNAME}"
+          export REGION="us-east-2"
+
+          echo '=== Cleaning up EKS cluster ${cluster} ==='
+          retry=3
+          while [ "${retry}" -gt 0 ]; do
+             eksctl delete cluster --name ${cluster} --region $REGION
+             if [ $? -eq 0 ]; then
+                break
+             fi
+             sleep 10
+             retry=$((retry-1))
+          done
+          if [ "${retry}" -eq 0 ]; then
+             echo "=== Failed to delete EKS cluster ${cluster}! ==="
+             exit 1
```

Comment:
Just checking that if we exit with code 1 here the job will report a failure? I think it's important that it does as we need to be notified so we can delete the cluster manually and not rack up a huge aws bill

Reply:
Right now the cluster cleanup is implemented as a follow-up job after the completion of eks-test in Jenkins, regardless of finish status. This is to be conformant with the other tests in CAPV. If you believe it makes more sense to integrate the cleanup in test-conformance-eks.sh let me make that change. I need to look more into how we can be notified with a cluster cleanup failure

Reply:
The upside for having cluster cleanup as a separate job is that email notification when job fails can be easily added